packages/okta-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/okta-react",
-  "version": "0.0.12",
+  "version": "0.0.13",
```

Comment:
Any reason we aren't bumping this to `0.1.0` like the others?

Reply:
I don't consider the user agent change to be a feature add or functionality add.  It's not technically a bug fix but still feels more like a patch and a minor change.